clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -49,13 +50,23 @@ let languageClientService: LanguageClientService;
 let outputChannel: vscode.OutputChannel;
 const API_VERSION: string = "1.0.0";
 
-function initialize() {
+function initialize(context: vscode.ExtensionContext) {
   // We need lazy initialization to be able to mock this for unit testing
-  const copyBooksDownloader = new CopybookDownloadService();
+  vscode.workspace.fs.createDirectory(context.globalStorageUri);
```

Comment:
who is creating the nested directories?

Reply:
This is for file watcher to work. creation of the directories handled at downloadCopybookContent#  zoweExplorerApi
        .getMvsApi(...)
        .getContents(...)